### PR TITLE
Fix incorrect compiler#pre-processor link in jp guide page

### PR DESCRIPTION
**Pull request by omiend**

### CODE REVIEW

**Comment by GianlucaGuarini** [comment]
@cognitom could you please have a quick look at it? Thanks

File: _config.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ react_mini_router:
 translations:
   es: v2.2.4
   fr: v2.5.0
-  ja: v3.13.1
+  ja: v3.13.2
```
Comment (k-kuwahara) [inline _config.yml]: Thank you for your PR !!
But, plz don't change here. Because the version of riot itself is `3.13.1` 🙇

Comment (omiend) [reply _config.yml]: Ooops!
I will send again pull request.

**Comment by omiend** [comment]
`_config.yml` should not be changed.
I will send again pull request.
thx.

**Pull request closed without merging**